lib/biokbase/catalog/db.py

```diff
@@ -164,11 +164,19 @@ def _initialize_mongo_client(self):
             if self.mongo_user and self.mongo_psswd:
                 # Connection string with authentication
                 mongo_client = MongoClient(
-                    f"mongodb://{self.mongo_user}:{self.mongo_psswd}@{self.mongo_host}/{self.mongo_db}?authMechanism={self.mongo_authMechanism}"
+                    self.mongo_host,
+                    username=self.mongo_user,
+                    password=self.mongo_psswd,
+                    authSource=self.mongo_db,
+                    authMechanism=self.mongo_auth_mechanism,
+                    retryWrites=self.mongo_retry_writes
                 )
```

Comment:
Did you manually retest this?

Reply:
I manually built the image and deployed it to CI. The logs seem fine to me.

Reply:
:+1: